src/classes/CRLP_RollupProcessor.cls

```diff
@@ -202,6 +236,15 @@ public inherited sharing class CRLP_RollupProcessor {
         return this;
     }
 
+    /**
+     * @description Specifies that external data should not be used at all with this processor. Necessary when
+     * external package is using this processor to calculate rollups in order to not recursively include external data
+     */
+    public CRLP_RollupProcessor withExcludeExternalData() {
```

Comment:
Assuming that "External Data" here refers to retrieving the rollup state, what do you think about one of these alternate method names:
* `withoutRetrieveExternalRollupStateData`
* `withoutExternalRollupStateData` 
* `withExcludeExternalRollupStateData`

Reply:
I like `withoutRetrieveExternalRollupStateData`, it reads a little better. I'll make the change